tests/unit/preprocessor/_time/test_time.py

```diff
@@ -486,6 +487,24 @@ def test_clip_timerange_single_year_4d(self):
                 coord_name
             )
 
+    def test_clip_timerange_start_date_invalid_isodate(self):
+        cube = self._create_cube(
+            [[[[0.0, 1.0]]]], [150.0], [[0.0, 365.0]], "standard"
+        )
+        with pytest.raises(isodate.isoerror.ISO8601Error) as exc:
```

Comment:
`pytest.raises` has a `match` keyword to check the exception string (see https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions), but feel free to ignore that comment

Reply:
ah I know the match= kwatg, cheers! meh, will leave it as is, if you don't mind, that'd be fine tuning a Trabant given how poorly those tests are (you see they using UnitTest classes :man_facepalming: )